scene/gui/texture_progress_bar.cpp

```diff
@@ -548,9 +553,11 @@ void TextureProgressBar::_notification(int p_what) {
 								p *= get_relative_center();
 								p += progress_offset;
 								p = p.floor();
```

Comment:
```suggestion
```
Please note that this line should be also removed to represent actual center position.